doc/source/help/index.rst

```diff
@@ -177,25 +177,17 @@ the problem!
 
 .. _irc:
 
-Go on Slack or IRC to ask a question
-------------------------------------
+Go on Slack to ask a question
+-----------------------------
 
 If you want a fast, interactive experience, you could try jumping into our Slack
-or IRC channels to get your questions answered in a chatroom style environment.
+to get your questions answered in a chatroom style environment.
 
 To join our slack channel you will need to request an invite by going to
 https://yt-project.org/development.html, click the "Join as @ Slack!" button, and
 fill out the form. You will get an invite as soon as an administrator approves
 your request.
 
-Alternatively you can go to our IRC channel, which does not require an
-invite. You don't even need to have any special IRC client in order to join the
-IRC channel.  We are the #yt channel on irc.freenode.net, but you can also
-connect using your web browser by going to https://yt-project.org/irc.html .
-There are usually 2-8 members of the user base and development team online, so
-you'll probably get your answers quickly.  Remember to bring the information
-from the :ref:`last step <isolate_and_document>`.
-
```

Comment:
Are we doing away with IRC?  I guess that makes sense, since slack is pretty widespread now.

Reply:
Well ... So nobody is really on the irc channel anymore. Also, freenode kind of imploded in a rather outrageous and horrifying way, and our channel probably doesn't even exist anymore. So I think we might be okay ditching it? That doesn't mean I'm not sad about it, just that I think we should emphasize where we *do* have a presence, y'know?

Reply:
Makes sense.